signalfx-yammer/pom.xml

```diff
@@ -57,6 +57,10 @@
       <artifactId>guava</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.apache.commons</groupId>
```

Comment:
There is an equivalent of StringUtils.isEmpty in guava.. Lets use that one and not add yet another dependency